### PR TITLE
OP-910 - Add print button for Operations in Main Menu \ Reports

### PR DESCRIPTION
**Pull request by nlazzer1**
See OP-910.

Add print button for operations in main menu Reports (ref https://github.com/informatici/openhospital-core/pull/693)

### CODE REVIEW

File: src/main/java/org/isf/stat/gui/OperationsListLauncher.java [hunk 1]
```diff
@@ -0,0 +1,136 @@
+package org.isf.stat.gui;
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/stat/gui/OperationsListLauncher.java:1]: Please add the standard license header to this file; thanks.

File: src/main/java/org/isf/stat/gui/report/OperationsList.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package org.isf.stat.gui.report;
```
[the remark this answers is not in the document]
Comment (nlazzer1) [reply src/main/java/org/isf/stat/gui/report/OperationsList.java:22]: Done

File: bundle/language_it.properties [hunk 3]
```diff
@@ -890,6 +890,7 @@ angal.menu.btn.opd
 angal.menu.btn.opdchart                                                                                = Report Visita
 angal.menu.btn.operation                                                                               = Interventi
 angal.menu.btn.operationtype                                                                           = Tipologie Intervento
+angal.menu.btn.operationlist                                                                           = Lista Interventi
```
Comment (mwithi) [inline bundle/language_it.properties:893]: There's no need to add new keys in the language other than "en". Please revert this change.

File: rpt/operationslist_es.properties [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#Generated by Eclipse Messages Editor (Eclipse Babel)
+
+Operation.Type = Operation / Tipología
```
Comment (mwithi) [inline rpt/operationslist_es.properties:3]: There's no need to add new keys in the language other than "en". Please revert this change.

File: rpt/operationslist_fr.properties [hunk 5]
```diff
@@ -0,0 +1,7 @@
+#Generated by Eclipse Messages Editor (Eclipse Babel)
+
+Operation.Type  = Typologie / Maladie
```
Comment (mwithi) [inline rpt/operationslist_fr.properties:3]: There's no need to add new keys in the language other than "en". Please revert this change.

File: rpt/operationslist_it.properties [hunk 6]
```diff
@@ -0,0 +1,7 @@
+#Generated by Eclipse Messages Editor (Eclipse Babel)
+
+Operation.Type = Tipologia
```
Comment (mwithi) [inline rpt/operationslist_it.properties:3]: There's no need to add new keys in the language other than "en". Please revert this change.

File: src/main/java/org/isf/stat/gui/OperationsListLauncher.java (hunk 1 above)
Comment (mwithi) [reply src/main/java/org/isf/stat/gui/OperationsListLauncher.java:22]: I cannot see the license header yet, has it been pushed?

File: src/main/java/org/isf/stat/gui/report/OperationsList.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (mwithi) [reply src/main/java/org/isf/stat/gui/report/OperationsList.java:22]: I cannot see the license header yet, has it been pushed?

**Review by mwithi — CHANGES_REQUESTED** [review]
LGTM. Some changes requested, then we'll test before merge.
Thanks a lot!

File: src/main/java/org/isf/stat/gui/OperationsListLauncher.java (hunk 1 above)
Comment (mwithi) [reply src/main/java/org/isf/stat/gui/OperationsListLauncher.java:22]: I can see the header on the remote branch, maybe it's a GitHub bug here

File: src/main/java/org/isf/stat/gui/report/OperationsList.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (mwithi) [reply src/main/java/org/isf/stat/gui/report/OperationsList.java:22]: I can see the header on the remote branch, maybe it's a GitHub bug here

File: bundle/language_it.properties [hunk 7]
```diff
@@ -941,6 +941,7 @@ angal.menu.newbill
 angal.menu.opd                                                                                         = Visita
 angal.menu.opdchart                                                                                    = Report Visita
 angal.menu.operation                                                                                   = Interventi
+angal.menu.operationlist                                                                               = Lista Interventi
```
Comment (mwithi) [inline bundle/language_it.properties:944]:
```suggestion
```

File: bundle/language_it.properties [hunk 8]
```diff
@@ -1369,6 +1370,8 @@ angal.stat.october
 angal.stat.opdattendance                                                                               = HMIS 105 - Presenze OPD
 angal.stat.opdbydiagnosis                                                                              = HMIS 105 - OPD per diagnosi
 angal.stat.opdreferrals                                                                                = HMIS 105 - OPD Trasferimenti
+angal.stat.operationlist                                                                               = Lista Interventi
+angal.stat.operationreport.title                                                                       = Rapporto Interventi
```
Comment (mwithi) [inline bundle/language_it.properties:1374]:
```suggestion
```

File: bundle/language_it.properties [hunk 9]
```diff
@@ -1386,6 +1389,8 @@ angal.stat.rundiseaseslistbytype.btn
 angal.stat.rundiseaseslistbytype.btn.key                                                               = P
 angal.stat.runexamslistreportbytype.btn                                                                = Ordina Lista Esami per Tipologia
 angal.stat.runexamslistreportbytype.btn.key                                                            = E
+angal.stat.runoperationslistbytype.btn                                                                 = Ordina Lista Interventi per Tipologia
+angal.stat.runoperationslistbytype.btn.key                                                             = I
```
Comment (mwithi) [inline bundle/language_it.properties:1393]:
```suggestion
```

**Review by mwithi — APPROVED** [review]
